### ISSUE TITLE
Using BODY_25 with demo

### ISSUE THREAD

**Issue by SleepNetwork** [CMU-Perceptual-Computing-Lab/openpose#782]
### Issue Summary
I have installed and run the OpenPoseDemo, but the only models downloaded were COCO and MPI. How would I go about using BODY_25? Will I need to install the program and change the code or is there an easier way to do this with the demo?

Version: 1.3.0 Windows

**Comment by gineshidalgo99** [comment]
You are right, we haven't uploaded version 1.4.0 into the Releases section (which includes BODY_25). We will do that within 1 month if no bug issues are found. Otherwise, you can download and compile it by yourself if you wanna use it already.
